### PR TITLE
Circle support

### PR DESCRIPTION
**Pull request by thisisdhaas**
Adds circle support for running tests

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,31 @@
+COVERAGE_ANNOTATION=coverage_annotations
+TEST_CMD=manage.py test orchestra beanstalk_dispatch
```
Comment (marcua) [inline Makefile:2]: probably fine if we don't call out the projects, so we can add more w/o having to update this.

File: circle.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+test:
+  override:
+    - make lint:
+        pwd: .
```
Comment (marcua) [inline circle.yml]: the `pwd`s are required?

File: example_project/example_project/orchestra_settings.py [hunk 3]
```diff
@@ -0,0 +1,156 @@
+"""
+Settings for the Orchestra app.
+
+Modify these according to your setup, then make sure they are included in your
+main settings file by adding::
+
+    from .orchestra_settings import setup_orchestra
```
Comment (marcua) [inline example_project/example_project/orchestra_settings.py:7]: why the leading `.`?

File: example_project/example_project/orchestra_settings.py [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+    setup_orchestra(__name__)
+
+to the bottom of the file.
```
Comment (marcua) [inline example_project/example_project/orchestra_settings.py]: bottom of `settings.py`?

File: example_project/example_project/orchestra_settings.py [hunk 5]
```diff
@@ -0,0 +1,156 @@
+"""
+Settings for the Orchestra app.
+
+Modify these according to your setup, then make sure they are included in your
+main settings file by adding::
```
Comment (marcua) [inline example_project/example_project/orchestra_settings.py:5]: double : ?

File: example_project/example_project/orchestra_settings.py [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+    from .orchestra_settings import setup_orchestra
+    setup_orchestra(__name__)
+
+to the bottom of the file.
+"""
+
+import sys
+import os
+
+# Add local orchestra to the system path: only for testing
```
Comment (marcua) [inline example_project/example_project/orchestra_settings.py]: should this be in the project's settings.py?

File: example_project/example_project/urls.py [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+    2. Add a URL to urlpatterns:  url(r'^$', Home.as_view(), name='home')
+Including another URLconf
+    1. Add an import:  from blog import urls as blog_urls
+    2. Add a URL to urlpatterns:  url(r'^blog/', include(blog_urls))
+"""
+from django.conf.urls import include, url
+from django.contrib.auth import views as auth_views
+
+urlpatterns = [
+    url(r'^orchestra/accounts/logout_then_login/$',
```
Comment (marcua) [inline example_project/example_project/urls.py]: add some sort of TODO or explanation of this

File: example_project/example_project/urls.py [hunk 8]
```diff
@@ -0,0 +1,32 @@
+"""example_project URL Configuration
```
Comment (marcua) [inline example_project/example_project/urls.py]: maybe clean up this docstring?

File: Makefile (hunk 1 above)
Comment (thisisdhaas) [inline Makefile:2]: Discussed and ignored.

File: example_project/example_project/orchestra_settings.py (hunk 3 above)
Comment (thisisdhaas) [inline example_project/example_project/orchestra_settings.py:7]: Discussed. Ignored.

File: example_project/example_project/orchestra_settings.py (hunk 5 above)
Comment (thisisdhaas) [inline example_project/example_project/orchestra_settings.py:5]: ReST artifact.

File: circle.yml (hunk 2 above)
Comment (thisisdhaas) [inline circle.yml]: I guess not: I'll take them out and see!

File: example_project/example_project/orchestra_settings.py (hunk 6 above)
Comment (thisisdhaas) [inline example_project/example_project/orchestra_settings.py]: moved it.

File: example_project/example_project/urls.py (hunk 7 above)
Comment (thisisdhaas) [inline example_project/example_project/urls.py]: Added.

File: example_project/example_project/urls.py (hunk 8 above)
Comment (thisisdhaas) [inline example_project/example_project/urls.py]: Cleaned.

**Comment by marcua** [comment]
The ReST artifact isn't cleaned up yet

**Comment by marcua** [comment]
LGTM if you address those two last comments---no need for another review

**Pull request merged**